vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -75,11 +79,13 @@ namespace configKeys {
     export const existingSharedPath = 'sharedExistingDotnetPath'
     export const proxyUrl = 'proxyUrl';
 }
+
 namespace commandKeys {
     export const acquire = 'acquire';
     export const acquireGlobalSDK = 'acquireGlobalSDK';
     export const acquireStatus = 'acquireStatus';
     export const uninstall = 'uninstall';
+    export const uninstallPublic = 'uninstallPublic'

```

Comment:
what does public refer to?

Reply:
public meaning users can use it and it shows up in the command palette

Reply:
So what is opposite of it? Are there installations done by extension which user cannot see?

Reply:
There are about 10 other commands / apis here that are non 'public', by this it means that only other extensions calling our extension apis can use them. They are not labeled with 'private'